docs/source/AdministratorGuide/Resources/computingelements.rst

```diff
@@ -135,6 +135,32 @@ A commented example follows::
      }
    }
 
+
+HTCondor Computing Element
```

Comment:
This whole section just duplicates the information from https://dirac.readthedocs.io/en/latest/AdministratorGuide/Configuration/ConfReference/Resources/Computing/index.html#htcondorce-parameters
which is also kind of here
https://dirac.readthedocs.io/en/latest/CodeDocumentation/Resources/Computing/HTCondorCEComputingElement.html
(but this was my fault)

So I would prefer if this isn't added again in a third place.

In addition, it also isn't similar to the CREAM example, because the CREAM example is for an individual CE and not for all HTCondorCEs as is the case here.